### ISSUE TITLE
Broken highlight in lambda functions

### ISSUE THREAD

**Issue by kotborealis** [jeff-hykin/better-cpp-syntax#604]
Checklist

- [x] This problem exists even with the setting `"C_Cpp.enhancedColorization": "Disabled"`

## The code with a problem is:
```cpp
    auto compare =
        [](
            const QualifiedName& a,
            const QualifiedName& b
        ) -> auto /* broken! */ /*
        // broken
        {
            return a.raw < b.raw;
        };
```

## It looks like:
![image](https://user-images.githubusercontent.com/6463851/192552494-e7b43a5a-0e3c-470e-94e9-6d6d6074794d.png)

Theme: Dark+ (default dark)

## It should look like:

Stuff after `->` and before the next `{` should be properly highlighted.

**Comment by kotborealis** [comment]
[This](https://github.com/jeff-hykin/better-cpp-syntax/blob/afda142bcd4052b45d78cba82d9939608631f613/main/main.rb#L2034) seems to be incorrect.

**Comment by jeff-hykin** [comment]
> [This](https://github.com/jeff-hykin/better-cpp-syntax/blob/afda142bcd4052b45d78cba82d9939608631f613/main/main.rb#L2034) seems to be incorrect.

Thanks for linking to a specific code chunk. Fixed and pushed in `v1.15.20`
